app/core/download.py

```diff
@@ -142,7 +142,9 @@ def create_query(ingest_cycle_start: str) -> str:
         '"Document Role", '
         'd.variant_name as "Document Variant", '
         'p.source_url as "Document Content URL", '
-        'd.document_type as "Document Type", '
+        "INITCAP(d.valid_metadata::json#>>'{"
```

Comment:
I'm assuming this returns NULL if the data isn't there?

Reply:
Correct :)